### PR TITLE
Update dual-stack.md

### PR DESCRIPTION
**Pull request by zhangx501**
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> Remember to delete this note before submitting your pull request.
>
> For pull requests on 1.17 Features: set Milestone to 1.17 and Base Branch to dev-1.17
>
> For pull requests on Chinese localization, set Base Branch to release-1.16
> Feel free to ask questions in #kubernetes-docs-zh
>
> For pull requests on Korean Localization: set Base Branch to dev-1.16-ko.\<latest team milestone>
>
> If you need Help on editing and submitting pull requests, visit:
> https://kubernetes.io/docs/contribute/start/#improve-existing-content.
>
> If you need Help on choosing which branch to use, visit:
> https://kubernetes.io/docs/contribute/start#choose-which-git-branch-to-use.
>^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
>

### CODE REVIEW

File: content/zh/docs/concepts/services-networking/dual-stack.md [hunk 1]
```diff
@@ -96,14 +88,12 @@ The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack
 <!--
 ## Enable IPv4/IPv6 dual-stack
 -->
-
 ## 启用 IPv4/IPv6 双栈
 
 <!--
-To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
+To enable IPv4/IPv6 dual-stack, enable the `IPv6DualStack` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for the relevant components of your cluster, and set dual-stack cluster network assignments:
```
Comment (RA489) [inline content/zh/docs/concepts/services-networking/dual-stack.md]: Feature gates links shows 404 here

File: content/zh/docs/concepts/services-networking/dual-stack.md [hunk 2]
```diff
@@ -96,14 +88,12 @@ The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack
...
 -->
-
-要启用 IPv4/IPv6 双栈，为集群的相关组件启用 `IPv6DualStack` [功能门](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/)，并且设置双栈的集群网络分配：
+要启用 IPv4/IPv6 双栈，为集群的相关组件启用 `IPv6DualStack` [功能门](/docs/reference/command-line-tools-reference/feature-gates/)，并且设置双栈的集群网络分配：
```
Comment (RA489) [inline content/zh/docs/concepts/services-networking/dual-stack.md]: 404

**Comment by zhangx501** [comment]
/assign @zhangxiaoyu-zidif
您好，问题已经改正过来了，请问可以合并吗？

**Pull request merged**